unit2-java/lesson06.rst

```diff
@@ -4,14 +4,18 @@ Unit 2 (Java) - Lesson 6 Learning Objectives
 Goals
 -----
 
+- Understand the role of inheritance in enable effective code resuse.
+- Effectively use Java's inheritance tools to build clean, reusable code.
```

Comment:
As phrased, this sounds like a repeat of the first objective.

Reply:
I rephrased the first goal to be, "Understand the role of object-oriented inheritance to enable effective code resuse."

The distinction between this goal and the second is that the first points toward the more broad, language-agnostic concept of inheritance, while the second refers to the Java-specific syntax and tools used to implement inheritance.

Is it more clear now?